packages/create-mechanic/function-templates/svgjs-image/dependencies.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {
+    "@mechanic-design/engine-svg": "^2.0.0-beta.7",
+    "@svgdotjs/svg.js" : "^2.7.1"
```

Comment:
Tried installing it, but it threw error: `No matching version found for @svgdotjs/svg.js@^2.7.1`. I see you only changed the package name, but not the wanted version. Is it the same one?

Reply:
Now it works!